Selenium.Axe/Selenium.Axe/AxeContextOptions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Newtonsoft.Json;
```

Comment:
`Context` and `Options` are the names of two arguments to `axe.run`; if we're calling the latter `AxeRunOptions`, would `AxeRunContext` be a more consistent name than `AxeContextOptions` for this class?